lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -86,6 +85,14 @@ class LineChartPainter extends AxisChartPainter<LineChartData>
       drawBetweenBarsArea(canvas, size, data, betweenBarsData);
```

Comment:
It should be drawn after background and grid and ...

Reply:
Nice catch. What else beside the grid should it cover? I think that's it.

Reply:
I think other things are well